src/main/java/com/raisetech/servicepractice/service/MaterialService.java

```diff
@@ -0,0 +1,31 @@
+package com.raisetech.servicepractice.service;
+
+import com.raisetech.servicepractice.entity.Material;
+import com.raisetech.servicepractice.exception.MaterialNotFoundException;
+import com.raisetech.servicepractice.mapper.MaterialMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MaterialService {
+    private final MaterialMapper materialMapper;
+
+    public MaterialService(MaterialMapper materialMapper){
+        this.materialMapper = materialMapper;
+    }
+
+    public List<Material> all(){
```

Comment:
メソッドの命名は動詞がメインのほうが直感的で良いかなと思います！
```suggestion
    public List<Material> findAll(){
```

Reply:
`materialService.all()`
`materialService.findAll()`
と見比べた時に、過不足なく処理の内容が分かる命名はどちらかと考えると納得しやすいかもですね。
(例えば、allだけだとdeleteかもしれませんし...)